src/lib.rs

```diff
@@ -222,8 +222,9 @@ pub use num_bigint::BigUint;
 pub use rand_core;
 pub use signature;
 
-pub mod algorithms;
+mod algorithms;
```

Comment:
Aah, ok. Maybe make the methods within explicitly `pub(crate)` for clarity?

Reply:
Yep. Sounds like a good idea. I'll change that and add a warning too.

Reply:
done